doc/tutorials/dune-package-management/pinning.md

```diff
@@ -2,8 +2,8 @@
 
 When Dune is looking up packages to lock, it uses the (pre)configured OCaml
 package repositories. However it is also possible to manually specify the
-sources for packages; for example, if the package is not released in a package
-repository. This is called "pinning."
+sources of packages; for example, if the package is not released in a package
+repository. This is called "pinning".
```

Comment:
I think this change was deliberately done by @christinerose?

This seems to be a difference between [american and british english]. Personally I prefer the full stop to be outside the quote (as this also matches the typographic conventions of most other western languages) but I will defer the decision to native speakers :)

Reply:
Cool, I'll defer this decision to @christinerose. I feel like I'm about to learn something about english (or at least american english).

Reply:
Bella prefers them outside, and since it is common in British English (and we adhere to British spelling), we should probably keep them on the outside 😱 ... I've changed our style guide. Thanks for bringing this up!

Reply:
```ocaml
let s = "thank you;;"
```